views/home.hbs

```diff
@@ -1,9 +1,9 @@
 
 <div class="select__language">
-  <a href="/en/places">
-  <div class="english">ENGLISH</div>
+  <a href="/en/content">
+    <div class="english">{{this.english}}</div>
```

Comment:
Is there a reason for `{{this.english}}` instead of just `{{english}}`

Reply:
umm .. not really didn't think about .. thanks